packages/build-utils/src/lambda.ts

```diff
@@ -39,31 +28,61 @@ interface GetLambdaOptionsFromFunctionOptions {
   config?: Pick<Config, 'functions'>;
 }
 
-export const FILES_SYMBOL = Symbol('files');
```

Comment:
We can't remove this until we remove the usage in PR #7390

Reply:
If you want to merge this PR before 7390, then you can update the existing function to read `lambda.files` instead of `lambda[FILES_SYMBOL]`

Reply:
This PR is also updating the usage to use `lambda.files` instead, so not a blocker.